app/src/androidTest/java/com/github/palFinderTeam/palfinder/meetups/activities/MeetupViewTest.kt

```diff
@@ -0,0 +1,71 @@
+package com.github.palFinderTeam.palfinder.meetups.activities
+
+import android.content.Intent
+import android.icu.util.Calendar
+import androidx.test.core.app.ActivityScenario
+import androidx.test.core.app.ApplicationProvider.getApplicationContext
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.palFinderTeam.palfinder.R
+import com.github.palFinderTeam.palfinder.meetups.MeetUp
+import com.github.palFinderTeam.palfinder.meetups.TempUser
+import com.github.palFinderTeam.palfinder.utils.Location
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class MeetupViewTest {
+    private var meetup: MeetUp? = null
+    private val eventName = "dummy1"
+    private val eventDescription = "dummy2"
+    @Before
+    fun init(){
+        val date1 = Calendar.getInstance()
+        val date2 = Calendar.getInstance()
+
+        meetup = MeetUp(
```

Comment:
maybe you can set different values for date1 and date2 and test if they propagate correctly on Meetup creation as well (as you did for the Event name and description)